bundles/redhat-marketplace/4.0.4/manifests/minio-directpv.clusterserviceversion.yaml

```diff
@@ -7,6 +7,8 @@ metadata:
     createdAt: "2023-05-12T20:45:18Z"
     operators.operatorframework.io/builder: operator-sdk-v1.28.0
     operators.operatorframework.io/project_layout: unknown
+    marketplace.openshift.io/remote-workflow: https://marketplace.redhat.com/en-us/operators/minio-directpv-operator-rhmp/pricing?utm_source=openshift_console
```

Comment:
@cniackz can we also add this annotation for `bundles/redhat-marketplace/4.0.3/manifests/minio-directpv.clusterserviceversion.yaml`?